docs/installation/setup-gce.rst

```diff
@@ -11,7 +11,7 @@ You'll need to setup at least two nodes.
 
 #. Create a new cloud server:
 
-   * Visit the `Google Cloud Console <https://console.cloud.google.com/>`_.
+   * Visit the `Google Cloud Console <https://console.cloud.google.com/start>`_.
```

Comment:
This is still resulting in a `404`. I've tried https://cloud.google.com/console locally which resulted in a successful link check build with the other change from below.